docs/developers/reference/api/linea-estimategas.mdx

```diff
@@ -11,12 +11,9 @@ import TabItem from '@theme/TabItem';
 
 :::info
 
-`linea_estimateGas` is currently not available on the Linea Sepolia testnet, but will be activated
-soon. **Mainnet activation is scheduled for July 30, 2024.** 
-
-It will only be fully compatible with endpoints using the Besu client — if it is unavailable, try 
-updating Besu. If you prefer to use Geth, `linea_estimateGas` is only available if you redirect
-your request through an Infura endpoint.
+The `linea_estimateGas` is only fully compatible with endpoints using the Besu client — if it is 
+unavailable, try updating Besu. If you prefer to use Geth, `linea_estimateGas` is only available if 
+you redirect your request through an Infura endpoint.
```

Comment:
We may need to refine this statement. For example, I don't think this method is available in the vanilla Besu client (Used in our Run a node instructions). I think it's only when running [linea-sequencer](https://github.com/Consensys/linea-sequencer?tab=readme-ov-file#quickstart---running-linea-besu-with-plugins).

Reply:
Good point. I have a ticket open to make these edits to the run a node instructions — will confirm with the team then update the run a node instructions and this para.